### ISSUE TITLE
Swappable weapons interact oddly with Offhand Weapon training

### ISSUE THREAD

**Issue by rpherbig** [rpherbig/dr-scripts#385]
It tries to keep swapping the weapon until it works. I don't really know *why* it works, but reducing the spam would be nice. And I'm afraid that it only accidentally works, so it could break at any time.

```
>
[combat-trainer: checking weapons as "Small Edged"!=Offhand Weapon]
[combat-trainer]>stow my spike
You put your spike in your kidskin baldric.
>
[combat-trainer]>get my throwing.spike
You get a steel throwing spike from inside your kidskin baldric.
>
[combat-trainer: unsupported weapon swap, please help us add this weapon]
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light blunt weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light edged weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light blunt weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light edged weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light blunt weapon.
>
* Apparently without direction or thought, a giant black leucro pounces, its mouth curled back in a savage snarl at you.  You block with a metal target shield.  
[You're nimbly balanced and in excellent position.]
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light edged weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light blunt weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light edged weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light blunt weapon.
>
You feel fully attuned to the mana streams again.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light edged weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light blunt weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light edged weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light blunt weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light edged weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light blunt weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light edged weapon.
>
[combat-trainer]>swap my spike
You deftly change your grip on your spike so it can be used as a light thrown and light blunt weapon.
>
[combat-trainer]>swap my spike
* Apparently without direction or thought, a giant black leucro pounces, its mouth curled back in a savage snarl at you.  You partially block with a metal target shield.  
[You're nimbly balanced and in excellent position.]
>
[combat-trainer]>swap
You deftly change your grip on your spike so it can be used as a light thrown and light edged weapon.
>
You move a steel throwing spike to your left hand.
>
[combat-trainer]>cast
You gesture at a giant black leucro.
A stream of dull golden light jumps from you to a giant black leucro, which warps into a spiraling force as it slams into it!
The black leucro is blasted back, reeling backwards in complete disarray from the blow!
Roundtime: 2 sec.
>
You feel fully rested.
You feel fully attuned to the mana streams again.
>
[combat-trainer]>attack left
< Moving in gracefully, you thrust a steel throwing spike at a giant black leucro.  A giant black leucro attempts to dodge, avoiding most of the blow.  The spike lands a good strike (3/23) that tears a small piece of skin from the left calf.
[You're nimbly balanced and overwhelming opponent.]
[Roundtime 3 sec.]
```

**Comment by rpherbig** [comment]
Should be fixed by 986b7118929110ff093f3bf07c9a6eebcf4b8eff once I merge #387

**Comment by rpherbig** [comment]
#387 was merged in 820df9275d230fc099c4d268bf901dceebb46c57
